<launch>
  <node pkg="teleop" type="teleop_joy" name="teleop_joypad">
    <param name="scale_angular" value="1.0"/>
    <param name="scale_linear_x" value="0.5"/>
    <param name="scale_linear_y" value="0.5"/>
    <param name="axis_deadman" value="4"/>
    <param name="axis_linear_x" value="1"/>
    <param name="axis_linear_y" value="0"/>
    <param name="axis_angular" value="3"/>
    <remap from="/teleop_joypad/cmd_vel" to="cmd_vel"/>
  </node>

  <node pkg="joy" type="joy_node" name="joystick"/>

</launch>
